ground/src/main/java/com/google/android/ground/ui/home/mapcontainer/MapCardUiData.kt

```diff
@@ -0,0 +1,10 @@
+package com.google.android.ground.ui.home.mapcontainer
+
+/** Data used to populate the Map cards (either an Loi card, or a Suggest Loi card). */
+data class MapCardUiData(
```

Comment:
Should this also contain a enum type field instead of using `loiId` being null as a proxy for suggest loi card?

Reply:
I actually ended up deciding to make MapCardUiData a sealed interface and then add a new LoiCardUiData which wraps the loi for the loi cards.